config/config/sc-config.yaml

```diff
@@ -297,6 +297,14 @@ elasticsearch:
   #   definition:
   #    users:
   #      - "Developer Name"
+  #  - mapping_name: kibana_user
+  #    definition:
+  #      backend_roles:
+  #        - "kibana_dev"
+  #  - mapping_name: all_access
+  #    definition:
+  #      backend_roles:
+  #        - "kibana_admin"
```

Comment:
Just to be clear - [`all_access`](https://github.com/opendistro-for-elasticsearch/security/blob/master/src/main/resources/static_config/static_roles.yml#L6) gives you permission to all APIs in elasticsearch and full access in kibana.